billboardd/tklockinterface.cc

```diff
@@ -0,0 +1,46 @@
+
+/**
+ * Billboard - Low Power Mode Standby Screen for the N9
+ * Webpage: http://thp.io/2012/billboard/
+ * Copyright (C) 2012, 2013, 2014 Thomas Perl <thp.io/about>
+ * Copyright 2015 Elliot Wolk <elliot.wolk@gmail.com>
```

Comment:
also please note the modified copyright header on the NEW files i added. dunno what youd prefer. i dont have any particular preferences {leave it like this, modify it to make it clear that this small thing is the only thing i did, put your original header back and just add an authors/contributors, whatever}